.github/workflows/verify-repo-tags.yml

```
@@ -0,0 +1,80 @@
+name: "Verify Repo Tags"
+
+on:
+  schedule:
+    - cron:  '0 7 * * *'
+  workflow_dispatch: {}
+
+jobs:
+  configure-github:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Verify Repo Tags
+        id: verify_repo_tags
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: |
+          output=$(bundle exec rake github:verify_repo_tags)
+
+          untagged_repos=$(echo "$output" | grep 'Untagged govuk repos:' -A 1 | tail -n +2)
+          falsely_tagged_repos=$(echo "$output" | grep 'Falsely tagged govuk repos:' -A 1 | tail -n +2)
+
+          echo "::set-output name=UNTAGGED_REPOS::$untagged_repos"
+          echo "::set-output name=FALSELY_TAGGED_REPOS::$falsely_tagged_repos"
```

Not sure if the above will work - it very well could, but the following method is [tried and tested](https://github.com/alphagov/govuk-user-reviewer/blob/39865181c8c95d54b9c4e91b7e120e3886429674/.github/workflows/review-users.yml#L28-L33):

```suggestion
          echo "untagged_repos=$untagged_repos" >> "$GITHUB_OUTPUT"
          echo "falsely_tagged_repos=$falsely_tagged_repos" >> "$GITHUB_OUTPUT"
```